luaui/Widgets/gui_top_bar.lua

```diff
@@ -647,8 +649,25 @@ local function updateWind()
 	end
 end
 
+-- return true if tidal speed is *relevant*, enough water in the world (>= 10%)
+local function checkTidalRelevant()
```

Comment:
This would ideally be Spring.Utilities.GetWaterFraction() or something like that

Reply:
Yeah, that would make a lot of sense

Reply:
I'll do another MR later which moves it there, and returns the two relevant numbers (water tiles / non-water tiles), ok?

Reply:
Check against master to see if the method wasn't replaced, that sounds good

Reply:
yes, it was changed to only check if the minimal ground height is < -20.. imho not a good thing, because even *one* small (deep) puddle on a huge map will trigger it, but it is definitely *correct* ;)